src/cli/ionic-config.ts

```diff
@@ -25,7 +30,10 @@ export async function readConfig(): Promise<TelemetryConfig> {
     };
 
     await writeConfig(config);
-  } else if (!!config && !config['tokens.telemetry'].match(UUID_REGEX)) {
+  } else if (
+    !!config &&
```

Comment:
@splitinfinities is there a reason I can't remove this check? Per [this condition](https://github.com/ionic-team/stencil/pull/3014/files#diff-e5b3f8482d62190b36ef5298593adece27d38744b93bb9d8f4f29fcfa2716965R26), this _should_ always return `true`

Reply:
I intended this to be a guard clause to protect against accessing config if it ended up not being defined. I think what you changed it to is sensible as well.